### PR TITLE
Sam/custom env

### PR DESCRIPTION
**Pull request by dragonrider0601**
Modifications to simulated world sd5.world

Whole bunch of lines added to the world file but that's apparently what happens when you save the world file in Gazebo.

I think I pulled it off with the Git conventions.

### CODE REVIEW

**Comment by TSC21** [comment]
@dragonrider0601 your SDF's need to be refactored. Check the CI output for the problems.

File: models/sd5/iris.sdf [hunk 1]
```diff
@@ -0,0 +1,543 @@
+<sdf version='1.6'>
```
Comment (TSC21) [inline models/sd5/iris.sdf:1]: Where is this different from the original IRIS? You know this is auto-generated by a xacro extension right?

**Review by TSC21 — CHANGES_REQUESTED** [review]
Looks good in general but I don't understand the need for the IRIS model under the `sd5` folder.
1. Can you please remove the leading `-` (minus signs) before single zeros? Will look cleaner that way.
2. Please address the problems reported by the schema verification script (check Travis CI log).

File: models/sd5/iris.sdf (hunk 1 above)
Comment (dragonrider0601) [reply models/sd5/iris.sdf:1]: Oh sorry I specified the wrong target branch

**Comment by dragonrider0601** [comment]
I meant to specify a different branch in another repo to create the pull request for but forgot to so by default it set px4:master as the target branch.

_My mistake._

**Comment by TSC21** [comment]
> I meant to specify a different branch in another repo to create the pull request for but forgot to so by default it set px4:master as the target branch.

@dragonrider0601 we are open to receive the SD5 model at our `sitl_gazebo` if that's your wish though.

**Pull request closed without merging**